### PR TITLE
To Do list: interactive list

### PR DESCRIPTION
**Pull request by Uchenna-ibute**
1.  Add a new JavaScript file and import it as a module:
it will contain methods related to the status updates (completed: true / false).

2. Add event listener to the checkbox (change).

3. Update items object's value for completed key upon user actions.

4. Implement a function for the "Clear all completed" button (use filter() method).

5. Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

**Review by AshakaE — APPROVED** [review]
Hi @Uchenna-ibute , 

### Highlights ✨ 

- Code base looks clean 👍 
- Pull request 👌 
- Git flow ✅ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Proceed to the next level 👏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/module/checker.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import updateLocalStorage from './localstorage.js';
+
+const checker = (te, bo, to) => {
```
Comment (gsmalik030) [inline src/module/checker.js:3]: I think it would be better if you use more descriptive name.

File: src/style.css [hunk 2]
```diff
@@ -1,107 +1,136 @@
...
-  font-size: 20px;
-  font-weight: bold;
+.it .white {
+  display: none;
 }
 
-#do-input {
-  color: #000;
+#trash {
+  display: none;
```
Comment (gsmalik030) [inline src/style.css:103]: you repeated this code couple of times. I think you if you use a class that would be better.

**Review by gsmalik030 — COMMENTED** [review]
You did a great job 👍 
I would suggest some changes that I mentioned in comments.

Well Done 💯

**Pull request merged**